src/adagrams.js

```diff
@@ -1,15 +1,106 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const ENGLISH_LETTERS = generateAlphabet();
+  const LETTERS_FREQUENCY = [
+    9, 2, 2, 4, 12, 2, 3, 2, 9, 1, 1, 4, 2, 6, 8, 2, 1, 6, 4, 6, 4, 2, 2, 1, 2,
+    1,
+  ];
+  const NUM_OF_DRAW_LETTERS = 10;
+
+  const lettersInHand = [];
+  let randomIndex;
+  while (lettersInHand.length < NUM_OF_DRAW_LETTERS) {
+    randomIndex = Math.floor(Math.random() * ENGLISH_LETTERS.length);
+
+    if (LETTERS_FREQUENCY[randomIndex]) {
+      lettersInHand.push(ENGLISH_LETTERS[randomIndex]);
+      LETTERS_FREQUENCY[randomIndex] -= 1;
+    } else continue;
+  }
+  // console.log(`lettersInHand=   ${lettersInHand}`);
+  return lettersInHand;
 };
 
+function generateAlphabet(capital = true) {
+  return [...Array(26)].map((_, i) =>
```

Comment:
This is an interesting approach to generating 26 characters! A more straight forward way that you could get the letters of the alphabet is to store them in a data structure. You could create an object that has the keys as the letters and the values as the frequency of the letters. Then you could iterate over the object's keys and values to generate a pool of all available letters to pick from. 

If you create a constant value with the letters and their corresponding values, you could organize this file by creating another file like constants.js and you could put the object in there and import it into adagrams.js

Totally optional, but sometimes folks like to pull big objects into another file to keep their method more concise. Here's an [example](https://stackoverflow.com/questions/61213322/how-to-import-object-to-another-file/61755282#61755282) of how to do it.